src/XrdPss/XrdPssCks.hh

```diff
@@ -78,7 +78,7 @@ struct csInfo
 
 csInfo *Find(const char *Name);
 
-static const int csMax = 4;
+static const int csMax = 5;
```

Comment:
Is this not one more than it should? The last entry is `csTab[3]`, no? Is there a sentinel at the end?

Reply:
Yes, I agree with the analysis above.

@osschar Could you please check this?

Reply:
The following two lines should have implied that csMax is the size of the table caTab[].
static const int csMax = 8;
csInfo           csTab[csMax];

So, we have a fixed sized table of 8 elements. The use of csLast relative to looking/manipulating csTab should imply that the table is not completely populated so, in fact, csLast points to the last valid entry in the table. 

This is the case here. The code supports up to 8 different checksums. The table is pre-populated with native checksums but one is allowed to add additional checksums beyond that and those will be added to the table in the standard way one does that during configuration processing.

Reply:
Yes, if you look at my edited response, while don't explicitly say so, I would increase to be the same between the two versions. Thanks!